stdlib/2and3/crypt.pyi

```diff
@@ -0,0 +1,18 @@
+import sys
+from typing import List, NamedTuple, Optional
+
+
+if sys.version_info >= (3, 3):
+    class _Method: ...
+
+    METHOD_CRYPT: _Method
+    METHOD_MD5: _Method
+    METHOD_SHA256: _Method
+    METHOD_SHA512: _Method
+
+    methods: List[_Method]
+
+    def mksalt(method: Optional[_Method] = ...) -> str: ...
+    def crypt(word: str, salt: Optional[str] = ...) -> str: ...
```

Comment:
Salt can also be a `_Method`, as of 3.3: "Changed in version 3.3: Accept crypt.METHOD_* values in addition to strings for salt."